deploy/docker_compose.dev.yml

```diff
@@ -24,8 +24,8 @@ services:
       context: .
       dockerfile: dev.dockerfile
       args:
-        - UID=${UID}
-        - GID=${GID}
+        - UID=${HOST_UID}
+        - GID=${HOST_GID}
```

Comment:
We should probably use `HOST_UID` and `HOST_GID` instead of `UID` and `GID` as they are read only identifier and might throw error for some os images.([see logs here.](https://github.com/Healthlane-Technologies/Zango/actions/runs/10177178934/job/28148070593))

Reply:
@DevilsAutumn Make sense, we can also raise a separate PR for this change including changes in every places like dev dockerfile, prod dockerfile and docs